apps/hash-api/src/integrations/linear/webhook.ts

```diff
@@ -45,9 +49,32 @@ export const linearWebhook: RequestHandler<{}, "ok", string> = async (
     return;
   }
 
-  const _payload = JSON.parse(req.body) as LinearWebhookPayload;
+  const payload = JSON.parse(req.body) as LinearWebhookPayload;
+
+  const temporalClient = await createTemporalClient(logger, {
+    host: getRequiredEnv("HASH_TEMPORAL_SERVER_HOST"),
+    port: parseInt(process.env.HASH_TEMPORAL_SERVER_PORT || "7233", 10),
+    namespace: "HASH",
+  });
 
-  // @todo trigger update to HASH entities based on the payload
+  if (
+    ["create", "update"].includes(payload.action) &&
+    ["Issue", "User"].includes(payload.type)
+  ) {
+    const workflow = `${payload.action}${payload.type}`;
+    await temporalClient.workflow.execute(workflow, {
+      taskQueue: "integration",
+      args: [
+        {
+          // @todo Use correct account IDs
+          actorId: "00000000-0000-0000-0000-000000000000",
+          ownedById: "00000000-0000-0000-0000-000000000000",
+          issue: (payload as LinearWebhookPayloadChange).data,
```

Comment:
It might be a user
```suggestion
          payload: (payload as LinearWebhookPayloadChange).data,
```

Reply:
Good point, thank you. Fixed it in [`87b9e19` (#2811)](https://github.com/hashintel/hash/pull/2811/commits/87b9e192eb1d6ffaff6d76c97faa8e101f626660)